kubernetes/annotations_and_labels.md

```diff
@@ -47,13 +47,24 @@ This page defines common annotations and labels we set in Kubernetes objects.
   by the operator to recognize which object it should reconcile. When set on
   nodes it is used to set that information in the status with the
   statusresource.
+- `OPERATOR.giantswarm.io/branch` - (informational) branch from which this
+  instance of the `OPERATOR` was built from.
```

Comment:
```suggestion
- `giantswarm.io/branch` - (informational) branch from which this
  code was built from.
```

Reply:
yeah I know it seems slightly redundant, but just `giantswarm.io/branch` reads weird, i.e. a branch of what

Reply:
I mean this is what this change in docs is supposed to explain right?

Also it's easier on tooling:

```
kubectl get deploy -n giantswarm -l "app=aws-operator,giantswarm.io/branch=legacy"
````

vs

```
kubectl get deploy -n giantswarm -l "app=aws-operator,aws-operator.giantswarm.io/branch=legacy"
````

For me the first one is more appealing.

Reply:
Oh yeah you can do:

```
kubectl get deploy -n giantswarm -l "aws-operator.giantswarm.io/branch=legacy"
```

But still I'd like to avoid dynamic label names if not necessary. This is not intuitive IMO.

Reply:
https://github.com/giantswarm/fmt/pull/100/files#r390885624

Reply:
Done